package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "crds-net",
+  "version": "1.0.0",
+  "description": "## Setup",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib"
+  },
+  "scripts": {
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run --record --key f9895842-b05f-494c-9010-b87b26527851"
```

Comment:
should that key be put into an environment variable?

Reply:
Yes it should. I think once we start running this in 'production' we will want to re-establish the project and get a new key anyways, so no big deal we exposed this one in this PR/commit.

Reply:
I have the contentful env variables stored in a cypress.env.json file that's ignored. Want me to move it there?